MODULE.bazel

```diff
@@ -9,6 +9,7 @@ bazel_dep(name = "rules_shell", version = "0.4.0")
 bazel_dep(name = "gazelle", version = "0.42.0")
 bazel_dep(name = "buildifier_prebuilt", version = "8.0.3")
 bazel_dep(name = "lcov", version = "2.3")
+bazel_dep(name = "platforms", version = "0.0.11")
```

Comment:
This is to be able to refer to `@platforms` in BUILD files. Without it getting

`ERROR: no such package '@@[unknown repo 'platforms' requested from @@]//os': The repository '@@[unknown repo 'platforms' requested from @@]' could not be resolved: No repository visible as '@platforms' from main repository. Was the repository introduced in WORKSPACE? The WORKSPACE file is disabled by default in Bazel 8 (late 2024) and will be removed in Bazel 9 (late 2025), please migrate to Bzlmod. See https://bazel.build/external/migration`